docs/src/pages/guides/localization/localization.md

```diff
@@ -13,59 +13,60 @@ import { createMuiTheme, ThemeProvider } from '@material-ui/core/styles';
 import { zhCN } from '@material-ui/core/locale';
 
 const theme = createMuiTheme({
-  palette: {
-    primary: { main: '#1976d2' },
-  },
-}, zhCN);
+	palette: {
```

Comment:
Please use the project guidelines. 

E.g here you’re using tabs instead of spaces.